graph/vertex/AttributeVertex.java

```diff
@@ -95,7 +85,7 @@
     /**
      * An vertex wrapper that sorts based on its value, then type -- as opposed to type, then value.
      */
-    interface Value<VALUE> extends AttributeVertex<VALUE> {
+    interface Sort<VALUE> extends AttributeVertex<VALUE> {
```

Comment:
Can we rename this to something less generic than Value? We only use this for sorting.